client/my-sites/site-settings/form-general.jsx

```diff
@@ -442,6 +444,46 @@ const FormGeneral = React.createClass( {
 		);
 	},
 
+	helloVoteOption() {
+		// Note that years and months below are zero indexed
+		const site = this.props.site;
+
+		if ( site.jetpack ) {
+			return null;
+		}
+
+		return (
+			<FormFieldset>
+				<legend>{ this.translate( 'US Voter Registration Form' ) }</legend>
+				<ul>
+					<li>
+						<FormLabel>
```

Comment:
What I wrote to collect analytics is ugly, but I think this would be worse because there are other ways a checkbox can change (keyboard events for example).  We may just have to do without it.